nengo/builder/tests/test_transforms.py

```diff
@@ -109,9 +110,213 @@ def test_convinc_2d(
     assert allclose(signals[y], y0)
 
 
+@pytest.mark.parametrize("channels_last", (True, False))
+@pytest.mark.parametrize("strides", [(1, 1), (2, 2), (3, 2)])
+@pytest.mark.parametrize("kernel_size", [(4, 5), (5, 4), (5, 5)])
+@pytest.mark.parametrize("padding", ("same", "valid"))
```

Comment:
Again, I'd like to have the full test available for anyone making changes to this convolution code (covering so many different cases helped me to track down bugs, since there are so many potential edge cases in the code). But if you want to reduce things for the tests that typically run, I'm OK with that.